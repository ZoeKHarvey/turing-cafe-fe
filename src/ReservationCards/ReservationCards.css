.cards {
  background: white;
  border: 2px solid teal;
  width: 175px;
}

button {
  background-color: teal;
  color: white;
  height: 5vh;
}

.cancel__btn {
  width: 100%;
}