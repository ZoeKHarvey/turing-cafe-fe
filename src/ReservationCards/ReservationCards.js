import React from 'react';

const ReservationCards= ({ name, date, time, numGuests,}) => {
  // const favoriteClass = isFavorite ? 'favorite' : 'card'
  return (
    <section>
      <h3>{name}</h3>
      <p>{date}</p>
      <p>{time}</p>
      <p>{numGuests}</p>
      <button>ðŸ—‘</button>
    </section>
  )
}

export default ReservationCards;