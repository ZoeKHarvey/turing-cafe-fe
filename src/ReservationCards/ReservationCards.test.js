import React from 'react';
import { shallow } from 'enzyme';
import ReservationCards from './ReservationCards';

describe('ReservationCards', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<ReservationCards
      name='Doggan'
      date='2020/09/02'
      time='3:00'
      number={4}
      key={10}
    />);

    expect(wrapper).toMatchSnapshot()
  })
})