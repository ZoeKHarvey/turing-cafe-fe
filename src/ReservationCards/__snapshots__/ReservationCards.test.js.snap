// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCards should match the snapshot with all data passed in correctly 1`] = `
<section>
  <h3>
    Doggan
  </h3>
  <p>
    2020/09/02
  </p>
  <p>
    3:00
  </p>
  <p />
  <button>
    ðŸ—‘
  </button>
</section>
`;
